.travis.yml

```diff
@@ -56,26 +73,29 @@ install:
   - if [ "$TRAVIS_OS_NAME" == "osx" ]; then python3 -m pip install virtualenv; fi
   - if [ "$TRAVIS_OS_NAME" == "osx" ]; then virtualenv ../venv -p python3; fi
   - if [ "$TRAVIS_OS_NAME" == "osx" ]; then source ../venv/bin/activate; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python -m pip install --upgrade setuptools; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python -m pip install --upgrade pytest flake8 "sphinx<3" sphinx_rtd_theme coverage codecov hypothesis attrs; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python -m pip install --upgrade mypy || true; fi
-script:
-  #windows build
-  - if [ "$TRAVIS_OS_NAME" == "windows" ]; then source .travis/runPycairo.sh; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python setup.py sdist; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python setup.py bdist; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python setup.py install --root=_root; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python setup.py install --root="$(pwd)"/_root_abs; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python setup.py bdist_wheel; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python setup.py install --root=_root_setup; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then if [[ "${TRAVIS_PYTHON_VERSION:0:4}" != "pypy" ]] ; then python -m pip install .; fi; fi
-  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then python -m sphinx -W -a -E -b html -n docs docs/_build; fi
+  - if [ "$TRAVIS_OS_NAME" == "windows" ]; then curl -sS -L https://aka.ms/nugetclidl -o $TMP/nuget.exe; fi
```

Comment:
using a directory in `$TMP` variable is lot better instead of directly temp directory. What do you say?

Reply:
Yes... *but* lets try and switch back to chocolatey and see if we can avoid this whole bit.